homeassistant/components/blebox/air_quality.py

```diff
@@ -0,0 +1,37 @@
+"""BleBox air quality entity."""
+
+from homeassistant.components.air_quality import AirQualityEntity
+
+from . import BleBoxEntity, create_blebox_entities
+from .const import DOMAIN, PRODUCT
+
+
+async def async_setup_entry(hass, config_entry, async_add):
+    """Set up a BleBox air quality entity."""
+    product = hass.data[DOMAIN][config_entry.entry_id][PRODUCT]
+    create_blebox_entities(product, async_add, BleBoxAirQualityEntity, "air_qualities")
+    return True
```

Comment:
```suggestion
```

Reply:
Fixed in all upcoming PRs: #35372 (air quality - this one), #35370 (lights) and 35373 (climate).

(Draft PR #35552 will afterwards only contain requested review changes to the already-merged sensors).

Next time I'll make sure to apply such changes to each upcoming PR immediately. (Instead of a draft PR).

Thanks!